### PR TITLE
Fix release step

### PR DESCRIPTION
**Pull request by hanjano**

### CODE REVIEW

File: .github/workflows/blockapi-ci.yml [hunk 1]
```diff
@@ -65,15 +65,65 @@ jobs:
       - name: Update version
         run: |
           VER=$(semantic-release print-version)
-          semantic-release version
-          git commit -m "Update to v${VER}" -a 
-          git tag -d "v${VER}"
-          git tag "v${VER}"
-          git status -v
-          git tag -n
+          if [ ! -z "$VER" ]; then
```
Comment (hanjano) [inline .github/workflows/blockapi-ci.yml:68]: The release should not happen, if version is not updated

File: .github/workflows/blockapi-ci.yml [hunk 2]
```diff
@@ -65,15 +65,65 @@ jobs:
...
+            git tag "v${VER}"
+          fi
+          [ ! -z "$VER" ]
 
       - name: Push changes
         uses: ad-m/github-push-action@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: ${{ github.ref }}
+          tags: true
```
Comment (hanjano) [inline .github/workflows/blockapi-ci.yml:81]: Push tags, also

File: .github/workflows/blockapi-ci.yml [hunk 3]
```diff
@@ -65,15 +65,65 @@ jobs:
...
+
+  release:
```
Comment (hanjano) [inline .github/workflows/blockapi-ci.yml:83]: Moved release step here, as push from GH token doesn't cause actions to run (prevents infinite looping, correct)

File: .github/workflows/blockapi-ci.yml [hunk 4]
```diff
@@ -65,15 +65,65 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs: version
+    concurrency: release
+    if: ${{ success() && github.ref == 'refs/heads/master' }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: 'master'
```
Comment (hanjano) [inline .github/workflows/blockapi-ci.yml:92]: Don't pull ref that triggered the workfow, pull HEAD of master instead

**Review by ladovina — APPROVED** [review]

**Pull request merged**